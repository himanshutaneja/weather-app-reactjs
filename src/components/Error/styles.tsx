import styled from 'styled-components';

export const ErrorContainer = styled.div`
  background-image: linear-gradient(#046f94, #3493ad);
  margin: 0 0 1px;
  margin: 20px;
  padding: 10px;
  border-radius: 5px;
  color: #fff;
`;
