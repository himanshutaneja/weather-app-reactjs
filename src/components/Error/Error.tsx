import { FC } from 'react';
import { ErrorContainer } from './styles';

interface ErrorProps {
  message: string;
}

const Error: FC<ErrorProps> = ({ message }) => {
  return (
    <ErrorContainer>
      <strong>Error. </strong>
      {message}
    </ErrorContainer>
  );
};

export default Error;
