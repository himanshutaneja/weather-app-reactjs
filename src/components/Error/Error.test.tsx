import { shallow, ShallowWrapper } from 'enzyme';
import Error from './Error';

describe('Error', () => {
  let wrapper: ShallowWrapper;
  beforeEach(() => {
    wrapper = shallow(<Error message="error message" />);
  });

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
