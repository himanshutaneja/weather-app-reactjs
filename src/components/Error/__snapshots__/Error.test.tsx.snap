// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should render correctly 1`] = `
<styled.div>
  <strong>
    Error. 
  </strong>
  error message
</styled.div>
`;
